src/dotnet/APIView/APIViewWeb/Repositories/DevopsArtifactRepository.cs

```diff
@@ -56,10 +56,25 @@ public async Task<Stream> DownloadPackageArtifact(string repoName, string buildI
 
         private async Task<string> getDownloadArtifactUrl(string buildId, string artifactName, string project)
         {
+            var pauseBetweenFailures = TimeSpan.FromSeconds(2);
```

Comment:
is this number arbritary? or it has history to it

Reply:
2 seconds should be ideally enough to start the delay span. So, it's just arbitrary.